src/cdk-experimental/menu/menu.ts

```diff
@@ -67,80 +66,52 @@ import {MENU_TRIGGER, MenuTrigger} from './menu-trigger';
     {provide: CDK_MENU, useExisting: CdkMenu},
   ],
 })
-export class CdkMenu extends CdkMenuGroup implements Menu, AfterContentInit, OnInit, OnDestroy {
-  /**
-   * Sets the aria-orientation attribute and determines where menus will be opened.
-   * Does not affect styling/layout.
-   */
-  @Input('cdkMenuOrientation') orientation: 'horizontal' | 'vertical' = 'vertical';
-
+export class CdkMenu extends CdkMenuBase implements AfterContentInit, OnDestroy {
   /** Event emitted when the menu is closed. */
   @Output() readonly closed: EventEmitter<void | 'click' | 'tab' | 'escape'> = new EventEmitter();
 
-  /** Handles keyboard events for the menu. */
-  private _keyManager: FocusKeyManager<CdkMenuItem>;
-
-  /** Manages items under mouse focus. */
-  private _pointerTracker?: PointerFocusTracker<CdkMenuItem>;
-
   /** List of nested CdkMenuGroup elements */
+  // TODO(mmalerba): Does it make sense for menubars to have groups as well?
```

Comment:
Calling out this TODO in case anyone has an idea why menu is different from menubar in this way

Reply:
Ok, I'll leave that out and if anyone ever requests it we can examine the use case and re-evaluate